app/models/authentication.rb

```diff
@@ -1,9 +1,10 @@
 class Authentication < ApplicationRecord
   acts_as_miq_taggable
   include_concern 'ImportExport'
-  include YAMLImportExportMixin
 
+  include YAMLImportExportMixin
```

Comment:
If this is being moved, can they be alphabetical?

Reply:
Sure. I've moved the other `include` statements so they're all together, too.